packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -135,6 +135,35 @@ describe('HttpValidator', () => {
         });
       });
     });
+
+    describe('path validation in enabled', () => {
+      describe('request is set', () => {
+        describe('request.path is set and has hyphens in path params', () => {
+          it('calls the path validator', () => {
+            validator.validateInput({
+              resource: {
+                method: 'get',
+                path: '/a-path/{a-id}/b/{b-id}',
+                id: '1',
+                request: {
+                  path: [
+                    { name: 'a-id', style: HttpParamStyles.Simple },
+                    { name: 'b-id', style: HttpParamStyles.Matrix },
+                  ],
+                },
+                responses: [{ code: '200' }],
+              },
+              element: { method: 'get', url: { path: '/a-path/1/b/;b=2' } },
```

Comment:
Is this even valid? Isn't matrix only possible for query strings?

Reply:
Yes Matrix parameters can be in the path. Also, this is basically the same test as the one above it that was already there. I just added a hyphen in the param names. 

BUT, now that I'm looking at this, I did need to update the `element` to use `;b-id=2` instead of `;b=2` to make this a real use case.

Updated in 066041f.